Dockerfile

```diff
@@ -36,15 +36,21 @@ WORKDIR /build/pufferpanel
 COPY go.mod go.sum ./
 RUN go mod download && go mod verify
 
+COPY client/package.json client/package-lock.json ./client/
+RUN cd client && npm ci
+
 COPY . .
 RUN ~/go/bin/swag init -o web/swagger -g web/loader.go && \
     go build -v -buildvcs=false -tags $tags -ldflags "-X 'github.com/pufferpanel/pufferpanel/v2.Hash=$sha' -X 'github.com/pufferpanel/pufferpanel/v2.Version=$version'" -o /pufferpanel/pufferpanel github.com/pufferpanel/pufferpanel/v2/cmd && \
     mv assets/email /pufferpanel/email && \
     cd client && \
-    npm install && \
     npm run build && \
     mv dist /pufferpanel/www/
 
+FROM builder AS dev
```

Comment:
If there aren't going to be used, I'd prefer they aren't referenced. This isn't something to leave in a production image

Reply:
AFAIK this doesn't go into the prod image (only /pufferpanel is copied from the builder stage),

Reply:
Then there isn't really a reason to include it?